.github/workflows/scan_for_secrets.yaml

```diff
@@ -0,0 +1,19 @@
+name: Scan for Secrets
+on:
+  pull_request:
+  push:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 9 * * *" # run daily at 4AM EST/5AM EDT/1AM PST/2AM PDT
```

Comment:
**nitpick:** you could consider putting the comment above so it fits in one line. Also extremely nitpicky but maybe add a space before and after the "/"s

Reply:
Done